lib/matplotlib/axes/_base.py

```diff
@@ -163,31 +163,31 @@ def _process_plot_format(fmt):
     except ValueError:
         pass  # No, not just a color.
 
+    errfmt = ("{!r} is neither a data key nor a valid format string ({})"
```

Comment:
```suggestion
    errfmt = ("{!r} is not a data key or a valid format string ({})"
```

Reply:
sure

Reply:
This is a bit of an Oxford comma thing, but I would tend to say neither...nor here.  It's correct and a bit more formal.

Reply:
I do prefer neither/nor, so I'll go back to that :)

Reply:
Sounds good. :) I'm not formal enough in my speaking it seems!